test/components/meta_test.dart

```diff
@@ -57,6 +67,29 @@ List<GeoJSONObject> featureAndCollection(GeometryObject geometry) {
 }
 
 main() {
+  test('propEach', () {
+    collection(pt).forEach((input) {
+      propEach(input, (prop, i) {
+        expect(prop, {'a': 1});
+        expect(i, 0);
+      });
+    });
+  });
```

Comment:
I know turf.js doesn't have one, but you could easily add a `Feature` test for `propEach` here too to ensure the implementation for that code path is as expected.

Reply:
added tests for feature and feature collection independently for both `propEach` as `featureEach`